ClearKeep/Scenes/Register/RegisterView.swift

```diff
@@ -103,7 +104,7 @@ private extension RegisterView {
 				}))
 			}
 	}
-	
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/d0f13a652105952d1236881c6d1745f906086aa3/ClearKeep/Scenes/Register/RegisterView.swift#L107"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>RegisterView.swift:107</code></td>
    </tr>
  </tbody>
</table>

Reply:
fix complete